docs/source/isaura.rst

```diff
@@ -1,5 +1,160 @@
+
+This city just computes tracks and extracts topology information from **deconvoluted hits**. Therefore, it is analogous to the final stage of :doc:`esmeralda`. Unlike the latter, the input hits correspond to the deconvoluted ones coming from :doc:`beersheba` (instead of :doc:`penthesilea`), allowing us to obtain a finer tracking of events.
+
+.. _Isaura input:
+
+Input
+-----
+
+ * ``/Run/events``
+ * ``/Run/runInfo``
+ * ``/DECO/Events``
+ * ``/DST/Events``
+
+.. _Isaura output:
+
+Output
+------
+
+ * ``/Tracking/Tracks``: tracking-related information of events (in particular: track energy (``energy``), track length (``length``), number of voxels (``numb_of_voxels``), number of hits (``numb_of_hits``), minimum and maximum position computed with the hits comprising the track (``x_min``, ``y_max``, ...), blobs position (``blob1_z``, ``blob2_x``,...) and energy (``eblob1``, ``eblob2``), energy of the hits shared by both blobs (``ovlp_blob_energy``), and voxel size (``vox_size_x``, ``vox_size_y``, ``vox_size_z``). Each row corresponds to a different track, specified among the others within an event with its ``trackID``
+ * ``/Summary/Events``: global information related to the event. Each row is one event
+ * ``/DST/Events``: copy of the point-like information (*kdst*) of events, output of :doc:`penthesilea`
+ * ``/Filters/hits_select``: flag to indicate if an event did not pass due to the lack of hits
+ * ``/Filters/topology_select``: flag that indicates whether the topology has been performed properly 
+ * ``MC info``: Monte Carlo information of events. Only if run number < 0
+
+.. _Isaura config:
+
+Config
+------
+
+Apart from the :ref:`Common arguments to every city`, the parameters to run *Isaura* are the following:
+
+.. list-table::
+   :widths: 50 100 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``vox_size``
+     - ``[float,float,float]``
+     - X, Y, and Z dimensions of the voxels used in the voxalization of the hits.
+
+   * - ``strict_vox_size``
+     - ``bool``
+     - Flag to indicate if the size of the voxels is forced to be exactly the values provided in the previous argument (*True*),
+
+       or, on the other hand, if they are allowed to change a bit for each track, aiming to optimize the voxelization process (*False*).
+
+   * - ``energy_threshold``
+     - ``float``
+     - If the energy of one of the original end-point voxels is smaller than this value,
+
+       the voxel will be dropped and its energy redistributed to the neighbours.
+
+   * - ``min_voxels``
+     - ``int``
+     - The voxel dropping procedure commented on ``energy_threshold`` can only happen if the number of voxels is larger than the value specified in this argument.
+
+   * - ``blob_radius``
+     - ``float``
+     - Radius of the blobs.
+
+   * - ``max_num_hits``
+     - ``int``
+     - Maximum number of hits for an event to be processed.
```

Comment:
What happens if an event has more hits than this value? Is it filtered out? If that's the case, you should add this information to the ``/Filters/hits_select`` documentation above (meaning, an event can be filtered out either if it has no hits, and if it has too many hits).